pandas/tests/frame/common.py

```diff
@@ -56,3 +58,18 @@ def zip_frames(frames: list[DataFrame], axis: int = 1) -> DataFrame:
         index = frames[0].index
         zipped = [f.loc[i, :] for i in index for f in frames]
         return DataFrame(zipped)
+
+
+OPTIONAL_INTS = st.lists(st.one_of(st.integers(), st.none()), max_size=10, min_size=3)
```

Comment:
can you move these to pandas/_testing/_hypothesis.py 

we also have these
```
pandas/tests/io/parser/test_parse_dates.py:from hypothesis import (
pandas/tests/io/parser/test_parse_dates.py:# Strategy for hypothesis
pandas/tests/io/parser/test_parse_dates.py:def _helper_hypothesis_delimited_date(call, date_string, **kwargs):
pandas/tests/io/parser/test_parse_dates.py:def test_hypothesis_delimited_date(date_format, dayfirst, delimiter, test_datetime):
pandas/tests/io/parser/test_parse_dates.py:        except_out_dateutil, result = _helper_hypothesis_delimited_date(
pandas/tests/io/parser/test_parse_dates.py:    except_in_dateutil, expected = _helper_hypothesis_delimited_date(
pandas/tests/tslibs/test_ccalendar.py:from hypothesis import (
pandas/tests/tseries/offsets/test_ticks.py:from hypothesis import (
pandas/tests/tseries/offsets/test_offsets_properties.py:from hypothesis import (
pandas/tests/tseries/offsets/test_offsets_properties.py:from hypothesis.errors import Flaky
pandas/tests/tseries/offsets/test_offsets_properties.py:from hypothesis.extra.dateutil import timezones as dateutil_timezones
pandas/tests/tseries/offsets/test_offsets_properties.py:from hypothesis.extra.pytz import timezones as pytz_timezones
pandas/tests/tseries/offsets/test_offsets_properties.py:        # indicate to hypothesis that this is not a valid test case
```

Reply:
e.g. any creation routines i want to move.

Reply:
@jreback Just want to make sure I'm clear on what you're asking for here. In addition to the helpers I added, you're asking me to move all hypothesis helpers into that *proposed* new `_hypothesis.py` testing module?

Reply:
yes